.gitignore

```diff
@@ -0,0 +1,20 @@
+# Cargo compiled files and executables
+**/target/
+
+# These are backup files generated by rustfmt
+**/*.rs.bk
+
+# Local chain databases (defalut location)
+**/chains/
+
+# The cache for chain data in container
+.local
+
+# The cache for docker container dependency
+/.cargo/config
```

Comment:
what container is that? I don't see any dockerfiles in the repo or the PR
If this is part of another PR can we add that there please?